dtde/config.py

```diff
@@ -139,8 +139,8 @@ def _merge_dict(self, config):
         if 'password' in config and config['password'] is not None:
             self.password = config['password']
 
-        if 'custom_script' in config and config['custom_script'] is not None:
-            self.custom_script = config['custom_script']
+        if 'custom_scripts' in config and config['custom_scripts'] not in [[None], None]:
```

Comment:
'custom_scripts' used several times. Should be a constant.

Reply:
I don't understand what this means.